ouroboros-network/ouroboros-network.cabal

```diff
@@ -202,3 +202,39 @@ test-suite tests
                        -fno-ignore-asserts
   if flag(ipv6)
     cpp-options:       -DOUROBOROS_NETWORK_IPV6
+
+executable test-cddl
```

Comment:
Can we turn this into a test suite? It's just a matter of changing the executable to do the tests we want when called with no arguments.